### PR TITLE
fix links on ldap chart

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

File: charts/ldap/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # README
 
-This helm chart installs the jenkins infrastructure ldap service (ldap.jenkins.io) and is designed to be working with the [jenkinsciinfra/openldap](https://github.com/jenkinsciinfra/openldap) docker image.
+This helm chart installs the jenkins infrastructure ldap service (ldap.jenkins.io) and is designed to be working with the [jenkins-infra/ldap](https://github.com/jenkins-infra/ldap) docker image.
```
Comment (timja) [inline charts/ldap/README.md:3]: the link seems to have been intended to be https://hub.docker.com/r/jenkinsciinfra/ldap not that there's any info there?

**Review by timja — APPROVED** [review]

File: charts/ldap/README.md (hunk 1 above)
Comment (halkeye) [reply charts/ldap/README.md:3]: oh maybe yea, i'll leave it over night and see if anyone else knows what it should be, right now its not working.

Comment (timja) [reply charts/ldap/README.md:3]: this is probably better anyway

Comment (dduportal) [reply charts/ldap/README.md:3]: I expect the link to DockerHub to be part of the image's readme, so this change here is good ✅

**Review by dduportal — APPROVED** [review]

**Pull request merged**